move-vm-support/src/ss58_address.rs

```diff
@@ -0,0 +1,114 @@
+//! SS58 address format converter for Substrate and Move accounts.
+//! Based on the Pontem solution.
+
+use anyhow::{anyhow, ensure, Result};
+use blake2::{Blake2b512, Digest};
+use move_core_types::account_address::AccountAddress;
+
+// Substrate address prefix
+// Read more: https://docs.substrate.io/reference/address-formats/
+const SS58_PREFIX: &[u8] = b"SS58PRE";
+
+// Public key length in bytes
+const PUB_KEY_LENGTH: usize = 32;
+
+// Checksum length in bytes
+const CHECK_SUM_LEN: usize = 2;
```

Comment:
nit :grinning:
https://en.wikipedia.org/wiki/Checksum
```suggestion
const CHECKSUM_LEN: usize = 2;
```

Reply:
Changed.